admin/c2cgeoportal_admin/static/theme.css

```diff
@@ -33,10 +29,15 @@
   color: initial;
 }
 
+.item-theme,
 .item-layergroup_treeitem {
   margin-bottom: 0;
 }
+.item-theme .well,
```

Comment:
two times .well ?

Reply:
yes, two times well, for well in themes and well in layergroup_treeitem